docs/output.rst

```diff
@@ -0,0 +1,37 @@
+.. _otp: https://otp.dkfz.de/otp/
+
+Output
+======
+
+There is not much to say about the standard output and error of Roddy, except that the information on which jobs are submitted is printed to standard output, while all other information is printed to standard error. This simplifies the parsing of the submission results by systems like otp_.
```

Comment:
Not many people know OTP/otp and it is not public yet right?

Reply:
Yes. It will get a public site at https://gitlab.com/otp-platform/otp, but only in the next days. But generally, here it does not really fit well. Removed